core/trino-main/src/main/java/io/trino/execution/QueryStateMachine.java

```diff
@@ -545,6 +545,13 @@ private QueryStats getQueryStats(Optional<StageInfo> rootStage)
 
                 processedInputDataSize += stageStats.getProcessedInputDataSize().toBytes();
                 processedInputPositions += stageStats.getProcessedInputPositions();
+
+                // Not to include the exchange operator stat to avoid the addition of build-side input data multiple times for broadcast join
+                rawInputDataSize -= stageStats.getRawNonPartitionedInputDataSize().toBytes();
```

Comment:
Some comments:
1. I think `rawInputDataSize` and `rawInputPositions` can be removed now (we now have physical and internal network stats).
2. We could split `processedInputDataSize` similarly (scan and network), but we shouldn't perform subtraction on existing metric